src/app/(app)/(home)/support/page.tsx

```
@@ -0,0 +1,681 @@
+                support.
+              </p>
+            </CardContent>
+          </Card>
+        </div>
+      </section>
+
+      {/* FAQ JSON-LD for SEO */}
+      <Script
+        id="faq-jsonld"
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(faqJsonLd) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)